src/main/java/com/microsoft/sqlserver/jdbc/SQLServerParameterMetaData.java

```diff
@@ -21,7 +21,7 @@
 
 
 /**
- * SQLServerParameterMetaData provides JDBC 3.0 meta data for prepared statement parameters.
+ * Provides JDBC 3.0 meta data for prepared statement parameters.
```

Comment:
We can remove 'JDBC 3.0' from here. Its simply metadata for PS params. It was done to track 3.0/4.0 API differences.